lib/scripts/mtw.js

```diff
@@ -90,11 +90,13 @@ export class ContentScript {
 
   filter(countedWords, translationProbability, userDefinedTranslations, userBlacklistedWords) {
     var blackListReg = new RegExp(userBlacklistedWords);
+    var punctuationReg = new RegExp(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g);
     var countedWordsList = this.shuffle(this.toList(countedWords, function(word, count) {
       return !!word && word.length >= 2 && // no words that are too short
         word !== '' && !/\d/.test(word) && // no empty words
         word.charAt(0) != word.charAt(0).toUpperCase() && // no proper nouns
         !blackListReg.test(word.toLowerCase()); // no blacklisted words
+        !punctuationReg.test(word.toLowerCase()); // no punctuation marks
```

Comment:
This is perfect for now. But could you add a comment in the code, saying that lines 98 and 99 could be optimized by merging the two regular expressions and testing only once?